docs/utilities/batch.md

```diff
@@ -0,0 +1,258 @@
+---
+title: SQS Batch Processing
+description: Utility
+---
+
+The SQS batch processing utility provides a way to handle partial failures when processing batches of messages from SQS.
+
+**Key Features**
+
+* Prevent successfully processed messages from being returned to SQS
+* A simple interface for individually processing messages from a batch
+
+**Background**
+
+When using SQS as a Lambda event source mapping, Lambda functions can be triggered with a batch of messages from SQS.
+
+If your function fails to process any message from the batch, the entire batch returns to your SQS queue, and your Lambda function will be triggered with the same batch again.
+
+With this utility, messages within a batch will be handled individually - only messages that were not successfully processed
+are returned to the queue.
```

Comment:
This can be one paragraph, the break between each sentence suggest they might be disconnected.